ttfunk.gemspec

```
@@ -50,6 +50,6 @@ Gem::Specification.new do |spec|
   }
 
   spec.required_ruby_version = '>= 2.7'
-  spec.add_runtime_dependency('bigdecimal', '~> 3.1')
+  spec.add_runtime_dependency('bigdecimal', '>= 3.1, < 5')
```

Let's just remove the upper bound, since this gem is infrequently updated:
```suggestion
  spec.add_runtime_dependency('bigdecimal', '>= 3.1')
```